modules/persistenceService/src/test/scala/subsPersistenceLayer/SubscriptionServicePersistenceLayerSpec.scala

```diff
@@ -0,0 +1,60 @@
+package subsPersistenceLayer
+
+import Base.BaseSpec
+import cats.effect.IO
+import config.{ApplicationConfig, ConnectionUrl, DriverName, PassWord, User => DbUser}
+import doobie.util.update.Update0
+import migrations.DbMigrations
+import org.scalatest.Outcome
+import utils.Types.{Id, Owner, PostSubscriptionData, Repository}
+import java.util.UUID
+import doobie.postgres.implicits._
+
+class SubscriptionServicePersistenceLayerSpec extends BaseSpec {
+  override def withFixture(test: NoArgTest): Outcome = {
+
+    val driverName    = DriverName(container.driverClassName)
+    val connectionUrl = ConnectionUrl(container.jdbcUrl)
+    val user          = DbUser(container.username)
+    val password      = PassWord(container.password)
+
+    DbMigrations
+      .migrate[IO](ApplicationConfig(driverName, connectionUrl, user, password))
+      .flatMap(_ => IO(test()))
+      .unsafeRunSync()
+  }
+
+  describe("SubscriptionServiceQuery") {
+    describe("get") {
+      describe("when called") {
+        it("should return a list of repositories to the user") {
+          val id    = Id(UUID.randomUUID())
+          val query = SubscriptionServiceQuery.get(id)
+          check(query)
```

Comment:
I see you are type checking the queries, good. But is there somewhere in the test where you actually exercise the database? If not yet, it'd be good to add it in a follow-up PR.

Reply:
Yeah this Pr was actually to get Feedback before the Pr before too big.
I will make all the changes now. Thank you